options.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google LLC
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     https://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudsqlconn
+
+import (
+	"golang.org/x/oauth2"
+	apiopt "google.golang.org/api/option"
```

Comment:
Why the named import here?

Reply:
The package name is just "option", which feels non-specific in a folder called `options.go` that (will) include a `DialOption` and `DialerOption` - so it felt more readable to clarify the package name a bit.